CONTRIBUTING.md

```diff
@@ -21,14 +21,12 @@ After your pull request is submitted, we will review it at as soon as possible.
 ## Release Checklist
 
 1. Create new branch, update version string in package.json and in openInStackBlitz.ts, run `pnpm install`
-2. Take the built package from code sandbox CI and test the changes on StackBlitz (optional)
+2. Take the Continuous Release build and test the changes on StackBlitz (optional)
```

Comment:
we could remove or modify the line `Ensure that the pnpm dev script is running at all times during development.` as it is no longer a requirement, just a way to run the docs app locally